File: scratch/find_css_rules.js
const fs = require('fs');

const content = fs.readFileSync('C:/Users/anasr/.gemini/antigravity/scratch/edusphere/public/style.css', 'utf8');
const lines = content.split('\n');

const search_terms = [/\.glass-panel/, /\.card-tilt/, /animation:/, /@keyframes/];
search_terms.forEach(term => {
    console.log(`--- Search for: ${term} ---`);
    let count = 0;
    lines.forEach((line, idx) => {
        if (term.test(line)) {
            console.log(`Line ${idx+1}: ${line.trim()}`);
            count++;
            if (count >= 15) {
                console.log('... (truncated)');
                // break in forEach can be emulated by returning or just letting it continue
            }
        }
    });
});

File: scratch/find_css_rules.py
import re

with open("C:/Users/anasr/.gemini/antigravity/scratch/edusphere/public/style.css", "r", encoding="utf-8") as f:
    lines = f.readlines()

search_terms = [r"\.glass-panel", r"\.card-tilt", r"animation:", r"@keyframes"]
for term in search_terms:
    print(f"--- Search for: {term} ---")
    pattern = re.compile(term)
    count = 0
    for idx, line in enumerate(lines):
        if pattern.search(line):
            print(f"Line {idx+1}: {line.strip()}")
            count += 1
            if count >= 10:
                print("... (truncated)")
                break
